recipe/meta.yaml

```diff
@@ -2,71 +2,67 @@
-    - m2-base     # [win]
-    - m2-bash     # [win]
-    - m2w64-binutils  # [win]
-    - m2-make     # [win]
-    # at present, this one is only on conda-forge
-    #    conda config --add channels conda-forge
-    - msinttypes        # [win and py<34]
+    build:
+        - posix  # [win]
+        - m2w64-binutils  # [win]
+        - msinttypes  # [win and py<34]
 
 test:
-  files:
-    - gb-18030-2000.ucm  # from http://source.icu-project.org/repos/icu/data/trunk/charset/data/ucm/gb-18030-2000.ucm
-    - de.txt          # from ICU's data folder
-  commands:
-    # Libraries
-    {% set libs = [
-       "libicudata",
-       "libicui18n",
-       "libicuio",
-       "libicutest",
-       "libicutu",
-       "libicuuc"
-     ] %}
-    {% for each_lib in libs %}
-    - test -f $PREFIX/lib/{{ each_lib }}.a                                                 # [unix]
-    - test -f $PREFIX/lib/{{ each_lib }}.{{ version }}.dylib                               # [osx]
-    - test -f $PREFIX/lib/{{ each_lib }}.so.{{ version }}                                  # [linux]
-    {% endfor %}
-
-    # CLI tests
-    - genbrk --help               # [unix]
-    - gencfu --help               # [unix]
-    - gencnval --help             # [unix]
-    - gendict --help              # [unix]
-    - icuinfo --help              # [unix]
-    - icu-config --help           # [unix]
-    - makeconv gb-18030-2000.ucm  # [unix]
-
-    # TODO: Windows tests
+    files:
+        # http://source.icu-project.org/repos/icu/data/trunk/charset/data/ucm/gb-18030-2000.ucm
+        - gb-18030-2000.ucm
+        - de.txt  # From ICU's data folder.
+    commands:
+        # Libraries.
+        {% set libs = [
+            "libicudata",
+            "libicui18n",
+            "libicuio",
+            "libicutest",
+            "libicutu",
+            "libicuuc"
+            ] %}
+        {% for each_lib in libs %}
+        - test -f $PREFIX/lib/{{ each_lib }}.a                    # [unix]
+        - test -f $PREFIX/lib/{{ each_lib }}.{{ version }}.dylib  # [osx]
+        - test -f $PREFIX/lib/{{ each_lib }}.so.{{ version }}     # [linux]
+        {% endfor %}
+        # CLI tests.
+        - genbrk --help               # [unix]
+        - gencfu --help               # [unix]
+        - gencnval --help             # [unix]
+        - gendict --help              # [unix]
+        - icuinfo --help              # [unix]
+        - icu-config --help           # [unix]
+        - makeconv gb-18030-2000.ucm  # [unix]
+        - conda inspect linkages -n _test icu  # [linux]
 
 about:
-  home: http://site.icu-project.org/
-  license: MIT
-  summary: International Components for Unicode
+    home: http://site.icu-project.org/
+    license: MIT
+    summary: 'International Components for Unicode.'
 
 extra:
-  recipe-maintainers:
-    - ccordoba12
-    - jakirkham
-    - msarahan
+    recipe-maintainers:
+        - ccordoba12
+        - jakirkham
+        - msarahan
+        - ocefpaf
```

Comment:
Can you add me to this please? I'm not sure what the deal is here, does it happen automatically, or is it up to the person themselves or to be done by someone already trusted with maintainence of the package?